README.md

```diff
@@ -210,6 +210,137 @@ This app uses [RSpec](http://rspec.info/) for Rails tests.
 
     rake parallel:spec
 
+## Environment variables used by the application
 
+### Sending emails
 
+#### To prisons
 
+We use MessageLabs to send emails to prisons because they're on the
+Government Secure Intranet (GSI). On preprod, we use Sendgrid to send
+to maildrop so we can get access to them easily. You will need to
+configure them if you want to send email when submitting a booking
+request locally.
+
+Source: `app/mailers/prison_mailer.rb`
+
+- `GSI_SMTP_HOSTNAME`
+- `GSI_SMTP_PORT`
+- `GSI_SMTP_DOMAIN`
+
+#### To visitors
+
+These SMTP settings are used to send emails to visitors. Because these using SendGrid's SMTP server, they must be sent via SendGrid. You will need to configured them if you want to try and test email by submiting a booking request locally.
+
+Source: `config/environments/production.rb`
+
+
+- `SMTP_HOSTNAME`
+- `SMTP_PORT`
+- `SMTP_DOMAIN`
+- `SMTP_USERNAME`
+- `SMTP_PASSWORD`
+
+### Change default value of
+
+#### The Service Url
+
+This URL is used to configure the session store host and to generate links in emails. For development and test environment is set to `localhost`.
+
+Source: `config/environments/production.rb` and `config/initializers/session_store.rb`
+
+- `SERVICE_URL`
+
+#### The Prison Data File
```

Comment:
I expect that this will be removed entirely when @tatyree moves the configuration to multiple files.